src/app/state/sign.rs

```diff
@@ -24,36 +27,30 @@ pub struct RevocationTransactionsTarget {
 
 impl RevocationTransactionsTarget {
     pub fn new(
```

Comment:
Please put a comment explaining why you're filtering on the cancel and unvault_emergency and not on the emergency, it took a while for me to understand 😅

Reply:
Fair :+1:

Reply:
Or maybe I should filter also on emergency for API consistency even if the method calling RevocationTransactionsTarget does not need to filter manager xpubs

Reply:
Right, I'd say either you filter the emergency or you use `stks_fingerprints` instead of `fingerprints` (at that point it's ok  not to filter the emergency)